x86/SearchMacros.asm

```diff
@@ -474,8 +509,19 @@ Display	2, "Search(alpha=%i1, beta=%i2,	depth=%i8, cutNode=%i9)	called%n"
 	      cmovl   r8d, eax
 		lea   ecx, [rdx-1]
 		xor   r9d, r9d
+		push r10
```

Comment:
push r10 and pop r10 can safely be deleted without changing the bench, but care must be taken in future patches not to inadvertently hammer r10 here